### PR TITLE
Create test.md

### PR DESCRIPTION
**Pull request by summershrimp**
<a href='https://github.com/tankxu'></a>

- [x] <a href='#crh-comment-Pull 00c01d894e8897bf95d31afbf780b6c14d38d0a8 test.md 1'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/2#discussion_r103163049'>File: test.md:L1</a></b>
- <a href='https://github.com/tankxu'></a> single comment
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/2#issuecomment-282667210'>General Comment</a></b>
- <a href='https://github.com/tankxu'></a> comment

<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?approve=0'></a>&nbsp;<a href='https://github.com/tankxu/test-ci/pull/2'></a>
<a href='#crh-end'></a>
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/tankxu/test-ci/pull/2%23issuecomment-282667210%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/2%23issuecomment-282667453%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/2%23issuecomment-284928247%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/2%23discussion_r103163049%22%2C%20%22https%3A//github.com/tankxu/test-ci/pull/2%23discussion_r103163357%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/2%23issuecomment-282667210%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22comment%22%2C%20%22created_at%22%3A%20%222017-02-27T09%3A19%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%22%2C%20%22created_at%22%3A%20%222017-02-27T09%3A20%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%2C%20%7B%22body%22%3A%20%22useless%22%2C%20%22created_at%22%3A%20%222017-03-08T02%3A34%3A33Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%2000c01d894e8897bf95d31afbf780b6c14d38d0a8%20test.md%201%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/tankxu/test-ci/pull/2%23discussion_r103163049%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22single%20comment%22%2C%20%22created_at%22%3A%20%222017-02-27T09%3A20%3A23Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A%2B1%3A%22%2C%20%22created_at%22%3A%20%222017-02-27T09%3A22%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/tankxu%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20test.md%3AL1%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/tankxu%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars0.githubusercontent.com/u/5106039%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/tankxu'></a>

- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/2#issuecomment-282667210'>General Comment</a></b>
- <a href='https://github.com/tankxu'></a> comment
- <a href='https://github.com/tankxu'></a> :shipit:
- <a href='https://github.com/tankxu'></a> useless
- [x] <a href='#crh-comment-Pull 00c01d894e8897bf95d31afbf780b6c14d38d0a8 test.md 1'></a> &nbsp;<b><a href='https://github.com/tankxu/test-ci/pull/2#discussion_r103163049'>File: test.md:L1</a></b>
- <a href='https://github.com/tankxu'></a> single comment

<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/tankxu/test-ci/pull/2?approve=0'></a>&nbsp;<a href='https://github.com/tankxu/test-ci/pull/2'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by tankxu** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/tankxu/test-ci/pull/2'>Refresh</a>.<a href='#crh-update-none'></a>

**Review by tankxu — COMMENTED** [review]
good

**Review by fatjyc — DISMISSED** [review]

**Review by tankxu — APPROVED** [review]
approve

**Review by summershrimp — COMMENTED** [review]
hahaha

**Review by vangie — DISMISSED** [review]

**Review by tankxu — DISMISSED** [review]
prefect

**Review by vangie — APPROVED** [review]

**Comment by tankxu** [comment]
comment

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# This is a test file
```
Comment (tankxu) [inline test.md:1]: single comment

**Comment by tankxu** [comment]
:shipit:

File: test.md (hunk 1 above)
Comment (tankxu) [inline test.md:1]: :+1:

**Comment by tankxu** [comment]
useless

**Pull request closed without merging**